wagtail/admin/views/collections.py

```diff
@@ -37,6 +51,7 @@ class Create(CreateView):
     edit_url_name = "wagtailadmin_collections:edit"
     index_url_name = "wagtailadmin_collections:index"
     header_icon = "folder-open-1"
+    _show_breadcrumbs = True
```

Comment:
Wouldn't need to manually set these if we had wrapped the views in a viewset, but I think that's a low priority for now so I opted for the quicker solution.